utils/ssh/connect.go

```diff
@@ -65,10 +65,7 @@ func (s *SSH) Connect(host string) (*ssh.Client, *ssh.Session, error) {
 
 	session, err := client.NewSession()
 	if err != nil {
-		err := client.Close()
```

Comment:
Why ignore the error?

Reply:
@fanux I think the top level error should be returned instead